### ISSUE TITLE
Couldn't find any package by glob 'dotnet-sdk-3.1'

### ISSUE THREAD

**Issue by wshaddix** [dotnet/docs#18084]
After following the docs for ubuntu 20.02 I get the error
```
Couldn't find any package by glob 'dotnet-sdk-3.1'
```

I then followed the docs for troubleshooting this issue, mainly running
```
sudo dpkg --purge packages-microsoft-prod && sudo dpkg -i packages-microsoft-prod.deb
```

and then tried to install dotnet-sdk-3.1 but I still get the exact same error

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 730afb93-8f52-44ea-a9df-ba30bdd1bfd4
* Version Independent ID: 57d51e3d-e664-44ee-5b98-5fd0af27fba0
* Content: [Install .NET Core on Ubuntu 20.04 package manager - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-2004#feedback)
* Content Source: [docs/core/install/linux-package-manager-ubuntu-2004.md](https://github.com/dotnet/docs/blob/master/docs/core/install/linux-package-manager-ubuntu-2004.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

**Comment by jk8260** [comment]
same

**Comment by lzuhuo** [comment]
Same too

**Comment by XorZy** [comment]
Same problem also

**Comment by CryptoRodeo** [comment]
Same here

**Comment by AllwinSamuelJeba** [comment]
Same problem 
E: Unable to locate package dotnet-sdk-3.1
E: Couldn't find any package by glob 'dotnet-sdk-3.1'
E: Couldn't find any package by regex 'dotnet-sdk-3.1'

**Comment by xSke** [comment]
The 20.04/focal repository doesn't contain the packages yet. Looks like it still only has placeholder packages: https://packages.microsoft.com/ubuntu/20.04/prod/pool/main/

**Comment by petrutaraul** [comment]
Same. I was happy to see that SDK is available for 20.04. I deleted Windows and I installed Ubuntu right away. Now I need to go back on Windows, because I want to finish my project. Coming on Ubuntu when will be useful.

**Comment by thebf** [comment]
I have the same issue though i was able to use the package from the snap store.

**Comment by CryptoRodeo** [comment]
I was also able to get it from the snap store, thanks for the heads up @thebf !

**Comment by thebf** [comment]
Never mind, the dotnet cli tools are not quite available so certain aspects of working with aspnet core is not smooth with the snap store approach

**Comment by dagood** [comment]
> The 20.04/focal repository doesn't contain the packages yet. Looks like it still only has placeholder packages: https://packages.microsoft.com/ubuntu/20.04/prod/pool/main/

This is right--there are are some workarounds in the dotnet/core issue about this: https://github.com/dotnet/core/issues/4360. I've asked there if the 20.04 page should be taken down (since we don't have packages on the repo yet) or we could potentially publish workaround instructions.

@Thraka @leecow @rbhanda /cc @NikolaMilosavljevic

**Comment by chgray** [comment]
same

**Comment by CryptoRodeo** [comment]
I was able to get dotnet working on Ubuntu using the .NET core script and then adding the path variables to both `~/.profile` and `~/.bash`
https://dotnet.microsoft.com/download/dotnet-core/scripts

**Comment by Thraka** [comment]
@dagood I thought it was released and ready? Was I wrong? I must have crossed some wires somewhere.

**Comment by torsilver** [comment]
> I have the same issue though i was able to use the package from the snap store.

真的能找到 非常感谢

**Comment by Thraka** [comment]
Removed the article for now. Once it's actually ready, I'll republish the article. Ack!

**Comment by dagood** [comment]
Thanks! For anyone who hits this thread later--workarounds and the status of getting .NET Core packages released for 20.04 are at https://github.com/dotnet/core/issues/4360.

**Comment by gansopro** [comment]
I typed these commands and it worked:

sudo apt-get install -y gpg
wget -O - https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o microsoft.asc.gpg
sudo mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/
wget https://packages.microsoft.com/config/ubuntu/22.04/prod.list
sudo mv prod.list /etc/apt/sources.list.d/microsoft-prod.list
sudo chown root:root /etc/apt/trusted.gpg.d/microsoft.asc.gpg
sudo chown root:root /etc/apt/sources.list.d/microsoft-prod.list
sudo apt-get update && \
  sudo apt-get install -y dotnet-sdk-3.1

But now I formatted my PC and I'm trying to install again but the error appears as if I didn't type the above commands.
